core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -1120,10 +1121,12 @@ created(#knm_phone_number{created = undefined}) -> kz_util:current_tstamp();
 created(#knm_phone_number{created = Created}) -> Created.
 
 -spec set_created(knm_phone_number(), gregorian_seconds()) -> knm_phone_number().
+set_created(PN=#knm_phone_number{created = undefined}, undefined) ->
```

Comment:
@fenollp these are stored as the string undefined, should have two function clauses here... one for the atom and one for the string.